test-values.yaml

```diff
@@ -143,25 +179,9 @@ accessControl:
         ]
       }
 
-jmx:
-  enabled: true
-  registryPort: 9080
-  serverPort: 9081
-  exporter:
-    enabled: true
-    image: bitnami/jmx-exporter:latest
-    pullPolicy: Always
-    port: 5556
-    configProperties: |
-      startDelaySeconds: 0
-      hostPort: 127.0.0.1:9080
-      rules:
-        - pattern: 'trino.memory*'
-        - pattern: 'trino.execution<name=QueryManager>*'
-
 serviceMonitor:
   enabled: true
-  selector:
+  labels:
```

Comment:
Why this change is necessary?

Reply:
I think this was a bug, the selector value is not used anywhere in the chart.

Reply:
Indeed, it's not used, but maybe we should just remove it then? I'm not sure a `prometheus:default` label is useful.